modules/nf-core/snpsift/split/tests/main.nf.test

```diff
@@ -11,84 +11,94 @@ nextflow_process {
                 { assert snapshot(
-					path(process.out.out_vcfs[0][1]).vcf.variantsMD5,
-					process.out.versions
-					).match()
-				}
+                    path(process.out.out_vcfs[0][1]).vcf.variantsMD5,
+                    process.out.versions
+                    ).match()
+                }
             )
         }
     }
 
     test("test-snpsift-split-gz") {
-
         when {
             process {
                 """
                 input[0] = [
                     [ id:'test', split:true ], // meta map
-				    file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/vcf/test.vcf.gz', checkIfExists: true)
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/vcf/test.vcf.gz', checkIfExists: true)
                 ]
-
                 """
             }
         }
-
         then {
             assertAll(
                 { assert process.success },
                 { assert snapshot(
-					path(process.out.out_vcfs[0][1]).vcf.variantsMD5,
-					process.out.versions
-					).match()
-				}
+                    path(process.out.out_vcfs[0][1]).vcf.variantsMD5,
+                    process.out.versions
+                    ).match()
+                }
             )
         }
     }
 
-    test("test-snpsift-join") {
-
+    test("test-snpsift-split-join") {
         when {
             process {
                 """
                 input[0] = [
                     [ id:'test', split:false ], // meta map
-				    [
+                    [
                         file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/vcf/test.vcf', checkIfExists: true),
-				        file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/vcf/test2.vcf', checkIfExists: true)
+                        file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/vcf/test2.vcf', checkIfExists: true)
                     ]
-				]
-
+                ]
                 """
             }
         }
-
         then {
             assertAll(
                 { assert process.success },
                 { assert snapshot(
-					path(process.out.out_vcfs[0][1]).vcf.variantsMD5,
-					process.out.versions
-					).match()
-				}
+                    path(process.out.out_vcfs[0][1]).vcf.variantsMD5,
+                    process.out.versions
+                    ).match()
+                }
             )
         }
     }
 
+    test("test-snpsift-split-base-stub") {
+        options "-stub"
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', split:true], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/vcf/test.rnaseq.vcf', checkIfExists: true)
+                ]
+                """
+            }
+        }
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
```

Comment:
This suggestion is to obtain the versions yml file directly written to the snapshot
```suggestion
                { assert snapshot(
                    process.out,
                    path(process.out.versions[0]).yaml
                ).match() }
```